upgrade-tester/README.md

```diff
@@ -0,0 +1,5 @@
+# Upgrade Tester
+
+The `upgrade-gitops-maker.sh` and `upgrade-tester.sh` tools are used together to execute upgrades across a series of releases. The former populates a gitops repo with the NNF releases, and the latter installs those releases.
+
+See [Upgrade Tester](https://nearnodeflash.github.io/dev/repo-guides/release-nnf-sw/upgrade-tester.md) in the NNF documentation for details.
```

Comment:
I assume this documentation is coming next?

Reply:
https://github.com/NearNodeFlash/NearNodeFlash.github.io/pull/266